src/content/chapter1_functions/lesson06_generic_functions/code.gleam

```diff
@@ -14,6 +14,8 @@ pub fn main() {
   io.debug(twice("Hello", exclaim))
 }
 
+// The function signature (below) states that the type of 
+// `value` must be the same in all 4 places:
```

Comment:
This reads awkwardly- how could we reword it?

"The name `value` refers to the same type multiple times" ?

Reply:
Good. I think that does it!

Reply:
Thanks for being patient with me as I stumble through learning git/github 😄 Feel free to correct me as needed. 

Also, you're doing an amazing job Louis!  You must be amazingly busy! I hope you can manage to take a few weeks off now and then to step out of the grind....that's said purely selfishly... wanting you in the game for the long haul! 😄

Reply:
I also like:

"The function type signature below calls for values of the same type to be supplied in all positions where (the type variable named) `value` appears."